src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__name {
+    font-weight: 500;
+    font-size: $font-size-m;
+    line-height: $line-h-l;
+    margin: 0 0 4px;
+  }
+
+  &__code-product {
```

Comment:
The BEM element name `__code-product` seems to be inconsistent. BEM element names typically do not include dashes. If 'code' and 'product' are two separate elements, they should be named accordingly, or if 'code-product' is a single element, it should be named without a dash (e.g., `__codeProduct`).